src/scripting/push_check.hpp

```diff
@@ -72,6 +76,21 @@ namespace lua_check_impl
 	template<typename T>
 	using remove_constref = std::remove_const_t<std::remove_reference_t<std::remove_const_t<T>>>;
 
+
+	template<typename T>
```

Comment:
This seems sorta useful, but is there any way to add a version of `lua_push` for it as well?